Bio/Wise/dnal.py

```diff
@@ -116,6 +117,7 @@ def __str__(self):
 
 
 def align(pair, match=_SCORE_MATCH, mismatch=_SCORE_MISMATCH, gap=_SCORE_GAP_START, extension=_SCORE_GAP_EXTENSION, **keywds):
```

Comment:
Can you break the line after ```extension=_SCORE_GAP_EXTENSION```? (To make it <80 char; we do not enforce this yet)

Reply:
That would be nice, but isn't essential for this pull request focused on the docstrings.